### ISSUE TITLE
Look into issues with Github linking and login flow for certain users

### ISSUE THREAD

**Issue by joshuagraber** [Police-Data-Accessibility-Project/data-sources-app#528]
### Context

Brief description. Is this part of a parent issue?

### Requirements

- Check out [this screen recording](https://drive.google.com/file/d/1WbJWi-ZInNWqyvc2airQcmr5n45rjNcV/view)
- @maxachis and @joshuagraber hypothesize that this is limited to staging users and is a relic of those users being wiped every day.
- Confirm that this is the case.
- Also: look into what it would take to preserve users in the lesser DBs. It would be nice to have stable testing users anyways. 

### Tests

- TBD

### Docs

- TBD

### Open questions

* Any uncertainties? Anything we're flexible about?

**Comment by maxachis** [comment]
To make sure I'm properly understanding the video, and to confirm with @joshuagraber on this:
1. Sign in with Github -> Get "You already have an account with this email address"
2. Sign in manually, sign-in successful. Follow-up `/profile` page states "Your account is linked with GitHub"
3. If accessing page via localhost, omitting the `linked=true` query parameter, user is given option to link account with GitHub.
4. Clicking this then successfully takes the user to a `linked=true` version of the profile page.
5. Signing out and then clicking `Sign in with GitHub` again produces the `You already have an account with this email address` error

**Comment by maxachis** [comment]
@joshuagraber Additionally, with a v2 production database setup which can persist for longer periods of time, I wonder if this is something we'll still observe in that environment.

**Comment by josh-chamberlain** [comment]
I agree that we can address this closer to launch. I linked to it from #248 so we don't forget.

**Comment by maxachis** [comment]
This is blocked pending #643

**Comment by josh-chamberlain** [comment]
This doesn't seem to be an issue any more. Not a launch blocker.
